### PR TITLE
Personalized Paths in recurring Canvases

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-3105**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
Hey @hessmjr, this is a draft of updates for the EA release of Personalized Paths in recurring Canvases. Could you check this over from a technical perspective and let me know if anything is missing that we should call out? I'd also like to add some images for the analytics view, if you have any test Canvases or images you could share with me, that would be super helpful! 🙏

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 1]
```diff
@@ -10,11 +10,20 @@ tool: Canvas
...
+## How Personalized Paths works
 
-## Prerequisites
+When Personalized Paths is turned on in an Experiment Path step, the behavior is slightly different depending on if your Canvas is set to send once or to recur:
 
-You can use Personalized Paths in your Experiment Step when your Canvas is set to send once. Personalized Paths is not available for recurring or triggered Canvases.
+- **Single-send Canvas:** A group of users is held back in a delay group. The remaining users pass into an initial test to train a look-alike model for a duration you configure—at least 24 hours for best results. After the test, a model is created to learn which user behaviors were associated with a greater likelihood of converting on a given path. Finally, each user in the delay group is sent down the path most likely to result in conversion for them based on the behaviors they exhibit and what the look-alike model learned during the initial test.
```
Comment (hessmjr) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:17]: Mark Rummell has been debating the `at least 24 hours for best results` aspect, but I don't feel its worth blocking this.

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 2]
```diff
@@ -10,11 +10,20 @@ tool: Canvas
...
+- **Recurring Canvas (early access):** An initial experiment is performed on all users who enter the Experiment Path during a specified window. After the experiment window, each user is sent down the path most likely to result in conversion for them.
```
Comment (hessmjr) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:18]: Should we note that if multiple sends before the experiment ends the user will continually be assigned their determined distribution? aka some to variant 1, some to variant 2, etc until a model is ready...winning path should be similar

**Review by hessmjr — APPROVED** [review]
Overall this looks great!  sorry for the delay

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:17]: Thanks for the heads up! If there's any concern, I'm also fine to remove this for now, and if anything we can add it back in if needed.

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md (hunk 2 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:18]: > user will continually be assigned their determined distribution

By this, do you mean that they'll always receive the same variant during the experiment window? That seems like a good thing to note, as it supports best practices for experimentation (i.e., keeping variables consistent). Would something like this be accurate? 

```suggestion
- **Recurring Canvas (early access):** An initial experiment is performed on all users who enter the Experiment Path during a specified window. To maintain the integrity of the experiment, if a user receives multiple messages before the window ends, they'll be assigned to the same variant each time. After the experiment window, each user is sent down the path most likely to result in conversion for them.
```

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md (hunk 1 above)
Comment (hessmjr) [reply _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:17]: If our docs say it elsewhere, such as Personalized Variant, then I say keep it.

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md (hunk 2 above)
Comment (hessmjr) [reply _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:18]: Yep, that is correct!

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:17]: Sounds good—we indeed do have this in PV docs so I'll keep it in. 

> We recommend 24 hours as a baseline to ensure statistically significant and meaningful results. The more time you allow for the test, the more responses will be received and the more Braze can optimize.

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 3]
```diff
@@ -46,15 +55,32 @@ This option will send future users down the mix of paths according to the percen
...
+{% endtab %}
+{% tab Recurring Canvas %}
+
+You can test up to four paths in a single Experiment Path. Add your paths and finish setting up your Canvas as needed, then launch it.  
+
+When the first user has entered the experiment, you can check the Canvas to see analytics as they come in and [track your experiment's performance]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/experiment_step/#tracking-performance).
+
+When the experiment window passes and the experiment is complete, all subsequent users to enter the Canvas will be sent down the path most likely to result in conversion for them.
+
+![]({% image_buster /assets/img/experiment_step/experiment_personalized_recurring_analytics.png %})
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:80]:
```suggestion
![]({% image_buster /assets/img/experiment_step/experiment_personalized_recurring_analytics.png %}){: style="max-width:75%;" }
```

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**